src/util.cpp

```diff
@@ -1581,3 +1580,13 @@ QString GetVersionAndNameStr ( const bool bWithHtml )
 
     return strVersionText;
 }
+
+QString MakeClientNameTitle ( QString win, QString client )
```

Comment:
Hm.  I thought we had client name first.  But we don't.  It's "window name clientname" when the client name is specified.  The change to the mixer window was to move the appname to the end -- but _not_ if clientname has been used.  Which is a bit weird but... this is following the same pattern, so fine :).